### PR TITLE
Itsm update

### PR DESCRIPTION
**Pull request by suehighmoor**

### CODE REVIEW

File: articles/cdsz/cdsz-ref-application-process.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ When you apply for a CDSZ service, you must provide the following documents to s
 
 If you want to use the CDSZ, you can initiate the application process in one of two ways:
 
-- Raise a Service Request through My Calls, specifying the relevant CDSZ product (UKCloud Guard, Walled Garden or Secure Remote Access). One of our Sales representatives, TAMs, or Cloud Architects will arrange a call to discuss your requirements and then close the Service Request.
+- Raise a Service Request through [My Calls](https://portal.ukcloud.com/support/ivanti), specifying the relevant CDSZ product (UKCloud Guard, Walled Garden or Secure Remote Access). One of our Sales representatives, TAMs, or Cloud Architects will arrange a call to discuss your requirements and then close the Service Request.
```
Comment (SteveHallUKCLoud) [inline articles/cdsz/cdsz-ref-application-process.md]: no longer provide UKCloud Guard

File: articles/cdsz/cdsz-ref-application-process.md [hunk 2]
```diff
@@ -129,7 +129,7 @@ The design review is facilitated by a Cloud Architect and will gather a clearer
 
 ## Stage 4: Application submission
 
-Raise a Service Request specifying the relevant CDSZ product (UKCloud Guard, Walled Garden or Secure Remote Access) and attach all the [required documents](#required-documents), as well as any review notes from the assurance wrap review. This request will then be assigned to a UKCloud representative to verify that all required information has been provided.
+Raise a Service Request, via the [My Calls](https://portal.ukcloud.com/support/ivanti) section of the UKCloud Portal, specifying the relevant CDSZ product (UKCloud Guard, Walled Garden or Secure Remote Access) and attach all the [required documents](#required-documents), as well as any review notes from the assurance wrap review. This request will then be assigned to a UKCloud representative to verify that all required information has been provided.
```
Comment (SteveHallUKCLoud) [inline articles/cdsz/cdsz-ref-application-process.md]: same again for UKCloud Guard

File: articles/cdsz/cdsz-ref-application-process.md (hunk 1 above)
Comment (suehighmoor) [reply articles/cdsz/cdsz-ref-application-process.md]: Removed ref to UKCloud Guard

File: articles/cdsz/cdsz-ref-application-process.md (hunk 2 above)
Comment (suehighmoor) [reply articles/cdsz/cdsz-ref-application-process.md]: Removed ref to UKCloud Guard

**Pull request merged**